frontend/src/exams/view/exam.ts

```diff
@@ -203,6 +203,10 @@ export interface TacticsRatingComponent {
 
     /** A description of the component to be displayed to users. */
     description: string;
+
+    /** Indicates rating was derived as a summary (e.g. average) of 0 or more exams
+     * taken within the dojo **/
+    examSummary?: boolean;
```

Comment:
I think it makes more sense to add a `link?: string;` attribute and then link to whatever it points to. In the future, we might want to point to different pages for different components, rather than hardcoding a link to the tests page. Also, this allows us to link directly to `/tests/tactics` or `/tests/checkmate` rather than just to `/tests`